helpers/PalladiumHelper.rb

```diff
@@ -18,16 +18,19 @@ def get_result_set_link
     "#{@palladium.result_set_id}"
   end
 
-  def add_result_and_log(example)
-    result = add_result(example)
+  def add_result_and_log(example, data_file = nil)
+    result = add_result(example, data_file)
     OnlyofficeLoggerHelper.log("Test is #{result['status']['name']}")
     OnlyofficeLoggerHelper.log(get_result_set_link)
   end
 
-  def add_result(example)
+  def add_result(example, data_file = nil)
     name = example.metadata[:description]
     status, comment = get_status(example)
-    @palladium.set_result(name: name, description: comment, status: status)
+    if data_file
+      comment = { "subdescriber": [{ "title": 'image size (byte)', "value": data_file }], "describer": [{ "title": 'comment', "value": comment }] }
+    end
+    @palladium.set_result(name: name, description: comment.to_json, status: status)
```

Comment:
Is `comment.to_json` needed?
Seems old variant works without it.
If it really needed - make it json iside if
```
if data_file
 comment = {"subdescriber"...}.to_json
```